models/public/open-closed-eye-0001/model.yml

```
@@ -14,11 +14,16 @@
 
 description: >-
   Fully convolutional network for recognition of eye state ('opened', 'closed').
-task_type: object_attributes
+task_type: classification
 files:
-  - name: FP32/opened-closed-eye-0001.xml
-    size: 37686
-    sha256: 702e374b7dc11b369c13c2f9c5fc638c617d19a9c625eba5fdac9d190fddba41
+  - name: open-closed-eye.onnx
+    size: 46164
+    sha256: 4daa100034482525a26c9afb9297c16580a531189e66e3d2b2ac7d32becfd593
     source: https://download.01.org/opencv/openvino_training_extensions/models/open_closed_eye/open_closed_eye.onnx
-framework: dldt
+framework: onnx
+model_optimizer_args:
+  - --mean_values=[127.0, 127.0, 127.0]
+  - --scale_values=[255, 255, 255]
+  - --output=19
+  - --input_model=$conv_dir/open_closed_eye.onnx
```

```suggestion
  - --input_model=$dl_dir/open_closed_eye.onnx
```

---

I would suggest
```suggestion
  - --input_model=$dl_dir/open-closed-eye.onnx
```
`converter.py` fails otherwise.